packages/store-ui/src/atoms/TextArea/stories/TextArea.stories.tsx

```diff
@@ -4,10 +4,11 @@ import type { Story, Meta, ArgTypes } from '@storybook/react'
 import Component from '../TextArea'
 import type { Props as TextAreaProps } from '../TextArea'
 import mdx from './TextArea.mdx'
+import type { ComponentArgTypes } from '../../../typings/utils'
 
-const TextAreaTemplate: Story<TextAreaProps> = (props) => {
+const TextAreaTemplate: Story<TextAreaProps> = ({ variant, ...props }) => {
```

Comment:
you never manipulate the prop variant, so u do not need spread that here.
```suggestion
const TextAreaTemplate: Story<TextAreaProps> = (props) => {
```